packages/create-bison-app/template/graphql/modules/user.ts.ejs

```diff
@@ -178,6 +178,6 @@ export const SignupInput = inputObjectType({
   definition: (t) => {
     t.string('email', { required: true });
     t.string('password', { required: true });
-    t.field('profile', { type: 'ProfileCreateOneWithoutUserInput' });
+    t.field('profile', { type: 'ProfileCreateNestedOneWithoutUserInput' });
```

Comment:
Another syntax change with latest Prisma, discovered through build error and tracking down type generations